test/libsolidity/semanticTests/various/code_access_runtime.sol

```diff
@@ -0,0 +1,25 @@
+contract D {
+    uint256 x;
+
+    constructor() public {
+        x = 7;
+    }
+
+    function f() public view returns (uint256) {
+        return x;
+    }
+}
+
+
+contract C {
+    function test() public returns (uint256) {
+	D d = new D();
+	bytes32 hash;
+	assembly { hash := extcodehash(d) }
+	assert(hash == keccak256(type(D).runtimeCode));
+	return 42;
```

Comment:
I'd say it doesn't hurt to have this test, even though I didn't end up implementing ``.runtimeCode`` in this PR.